src/qengine/opencl.cpp

```diff
@@ -563,37 +567,68 @@ void QEngineOCL::Apply2x2(bitCapInt offset1, bitCapInt offset2, const complex* m
     DISPATCH_TEMP_WRITE(waitVec, *powersBuffer, sizeof(bitCapInt) * bitCount, qPowersSorted, writeControlsEvent);
 
     // We load the appropriate kernel, that does/doesn't CALCULATE the norm, and does/doesn't APPLY the norm.
-    OCLAPI api_call;
+    char kernelMask = APPLY2X2_DEFAULT;
```

Comment:
Unsigned?

Reply:
I wondered, too, but unnecessary? The macros are hex, less than 8 bits.

Reply:
In any case, if it were anything else in Qrack. It would be unsigned, so fair enough.
Done.